pkg/minikube/registry/registry_test.go

```diff
@@ -22,6 +22,20 @@ import (
 	"k8s.io/minikube/pkg/minikube/config"
 )
 
+func TestDriverString(t *testing.T) {
```

Comment:
I don't think we need to test `String()`.

Reply:
It's functions like each other. I'm happy to remove this, but could you please explain why? For sure it will decrease code coverage.

Reply:
true! let's keep it :)